kategory/src/test/kotlin/kategory/data/MonoidKTests.kt

```diff
@@ -0,0 +1,26 @@
+package kategory
+
+import io.kotlintest.KTestJUnitRunner
+import org.junit.runner.RunWith
+
+@RunWith(KTestJUnitRunner::class)
+class MonoidKTests : UnitSpec() {
```

Comment:
I think this just needs to be removed since we never test typeclasses directly but through their instances